contracts/MarketOracle.sol

```diff
@@ -75,6 +78,7 @@ contract MedianOracle is Ownable, IOracle {
         minimumProviders = minimumProviders_;
     }
 
+    // payload is expected to be 18 decimal fixed point number.
```

Comment:
Let's keep to the NatSpec convention, especially for external functions.

https://solidity.readthedocs.io/en/develop/natspec-format.html

Reply:
look good now?